### PR TITLE
add support for redirect and html banners

### PR DESCRIPTION
**Pull request by rahulcs**

### CODE REVIEW

**Comment by rahulcs** [comment]
@floydpraveen 
@amarts please review.

File: unbxdSearch.js [hunk 1]
```diff
@@ -1560,12 +1560,16 @@ var unbxdSearchInit = function(jQuery, Handlebars){
 
       bannersToDraw = banner.banners.slice(0, this.options.bannerCount)
 	.reduce(function(prev, curr){
-	  return prev.concat(this.compiledBannerTemp(
-	    {
-	      landingUrl: curr.landingUrl,
-	      imageUrl: curr.imageUrl
-	    }
-	  ));
+	  if(curr.hasOwnProperty('bannerHtml')) {
```
Comment (floydpraveen) [inline unbxdSearch.js]: need for null check here. bannerHtml will be null when imageUrl is given

File: unbxdSearch.js [hunk 2]
```diff
@@ -1304,6 +1304,11 @@ var unbxdSearchInit = function(jQuery, Handlebars){
 
       this.currentNumberOfProducts = 0;
 
+      if(obj.hasOwnProperty('redirect')) {
+	window.location = obj['redirect']['value'];
+	return false;
+      }
```
Comment (amarts) [inline unbxdSearch.js]: is indentation 1 column?

**Comment by floydpraveen** [comment]
Don't know why, my comment is shown as comment on outdated diff cc @rahulcs @amarts

**Comment by amarts** [comment]
@floydpraveen thats because he made some changes and re-pushed it.

**Comment by floydpraveen** [comment]
@amarts no, after my comment there is no commit I could see, now same with your comment

**Comment by rahulcs** [comment]
@floydpraveen redirects will work after https://unbxddev.atlassian.net/browse/SRCH-436 is fixed.

File: unbxdSearch.js (hunk 1 above)
Comment (rahulcs) [inline unbxdSearch.js]: @floydpraveen added.

**Pull request merged**